awx/settings/defaults.py

```diff
@@ -1061,5 +1061,7 @@
 # system username for django-ansible-base
 SYSTEM_USERNAME = None
 
+# OIDC access settings
+DEFAULT_OIDC_ENDPOINT = 'https://sso.redhat.com/auth/realms/redhat-external'
```

Comment:
this probably shouldn't be a setting. I think we can hardcode it in this file, or in the analytics_proxy.py file, whichever you think is best